kafka-native-test-container/src/test/java/com/ozangunalp/kafka/test/container/KerberosContainer.java

```diff
@@ -36,15 +35,15 @@ public void createTestPrincipals() {
                     "addprinc -randkey client/localhost@EXAMPLE.COM");
             lsResult = execInContainer("kadmin.local", "-q",
                     "ktadd -norandkey -k /client.keytab client/localhost@EXAMPLE.COM");
-            copyFileFromContainer("/kafkabroker.keytab", "src/test/resources/kerberos/kafkabroker.keytab");
-            copyFileFromContainer("/client.keytab", "src/test/resources/kerberos/client.keytab");
+            copyFileFromContainer("/kafkabroker.keytab", "target/test-classes/kerberos/kafkabroker.keytab");
```

Comment:
just checking my mental model. this part is unrelated to the test instability.  you are just preventing the test run dirtying the source tree.

Reply:
Absolutely